### PR TITLE
Undoes skyrat 9mm nerf

### PR DESCRIPTION
**Pull request by Redrover1760**
## About The Pull Request

9mm is not a bad chamber in tg. But everyone always ditched C-CKs and I've never seen a Makarov in my life on this server.

https://github.com/Bubberstation/Bubberstation/pull/1400

As you can see in this PR, the intention of Makarov to have 30 damage when based around the other changes for C-CKs ammo. Instead, skyrat added an override for only AP and normal 9mm (Neglecting Hollow Point, which still does 40 damage) to be at 20 damage and 25 damage respectively.

Modularly reverts 9mm AP and 9mm Standard ammo to 27 damage for AP, and 30 damage for 9mm.

Tested on local.

## Why It's Good For The Game

This fully Cucks the C-CKS ammo compared to Wespes/any Carwo gun ever, and trashes the already under-performing makarov ammo with its incredibly limited 8 round magazines in the first place. 8 rounds to kill an armored individual with basic sec vest in a 9 round gun is pretty shit.

## Changelog

:cl:
balance: Reverts 9x25mm and 9x25mm AP skyrat nerf to tg values.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/aesthetics/guns/code/guns.dm [hunk 1]
```diff
@@ -482,11 +482,12 @@
 
 /obj/projectile/bullet/c9mm
 	name = "9x25mm bullet"
-	damage = 25
+//	damage = 25
```
Comment (StrangeWeirdKitten) [inline modular_skyrat/modules/aesthetics/guns/code/guns.dm:485]:
```suggestion
//	damage = 25 - BUBBER EDIT
```

File: modular_skyrat/modules/aesthetics/guns/code/guns.dm [hunk 2]
```diff
@@ -482,11 +482,12 @@
...
 
 /obj/projectile/bullet/c9mm/ap
 	name = "9x25mm armor-piercing bullet"
-	damage = 20
+//	damage = 20
+// BUBBER EDIT
```
Comment (StrangeWeirdKitten) [inline modular_skyrat/modules/aesthetics/guns/code/guns.dm:490]:
```suggestion
//	damage = 20 - BUBBER EDIT
```

**Review by StrangeWeirdKitten — CHANGES_REQUESTED** [review]

**Comment by StrangeWeirdKitten** [comment]
I'll let this PR sit for a little for other eyes to look over.

**Comment by projectkepler-ru** [comment]
i never neglected the hollow point, it was left as that as the armour weakness is pretty severe, the C-CK pistol is not my content and not my code responsibility

File: modular_skyrat/modules/aesthetics/guns/code/guns.dm
Comment (projectkepler-ru) [inline modular_skyrat/modules/aesthetics/guns/code/guns.dm:1]: Override this is a separate modular bubber folder

**Review by projectkepler-ru — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/aesthetics/guns/code/guns.dm
Comment (StrangeWeirdKitten) [reply modular_skyrat/modules/aesthetics/guns/code/guns.dm:1]: I told them to comment it out.

**Review by StrangeWeirdKitten — APPROVED** [review]

**Pull request merged**